compiler/src/dmd/hdrgen.d

```diff
@@ -1968,6 +1968,10 @@ private void visitVarDecl(VarDeclaration v, bool anywritten, ref OutBuffer buf,
             v._init.initializerToBuffer(buf, &hgs);
     }
 
+    const commentIt = hgs.importcHdr && isSpecialCName(v.ident);
+    if (commentIt)
+        buf.writestring("/+");
```

Comment:
shouldn't commented out declarations out be emitted only if `global.params.dihdr.fullOutput` is set?

Reply:
The `hgs.importcHdr` fulfills that role.